content/api/commands/writefile.md

```diff
@@ -30,7 +30,7 @@ default `cypress.json`).
 
 **<Icon name="angle-right"></Icon> contents** **_(String, Array, Object or Buffer)_**
 
-The contents to be written to the file.
+The contents to be written to the file. Contents must be no larger than 100MB in size.
```

Comment:
Is 100MB a hard limit or if the timeout is increased can the size increase?

Reply:
100MB is a hard limit based on our [socket server configuration](https://github.com/cypress-io/cypress/blob/develop/packages/socket/lib/socket.ts#L6). Anything greater than that will cause a socket transport error. I logged [another issue](https://github.com/cypress-io/cypress/issues/19142) to add better error reporting when that happens.